packages/shared/src/utils/config/platform-config.ts

```diff
@@ -28,7 +28,6 @@ export const platforms: TPlatforms = {
         is_hard_redirect: true,
         platform_name: 'Deriv GO',
         route_to_path: '',
-        url: 'https://app.deriv.com/redirect/derivgo',
```

Comment:
Why are you only removing the url?

Reply:
The URL was a value I added to the existing config. I basically reverted the changes that I did to implement the Back to DerivGo button. Also I didn't remove the confif completely because the `platform_name` & `hard_redirect` is being used in `common-store`